tests/pretty_format_yaml_test.py

```diff
@@ -45,3 +47,16 @@ def test_pretty_format_yaml_autofix(tmpdir):
     # file was formatted (shouldn't trigger linter again)
     ret = pretty_format_yaml([srcfile.strpath])
     assert ret == 0
+
+
+def test_pretty_format_multidoc_yaml_autofix(tmpdir):
```

Comment:
Nit (not-blocking): The new test can be replaces with a fixture on `test_pretty_format_yaml_autofix` so we can avoid duplicating code.

Example
```python
@pytest.fixture('no_pretty_file_name', ['not-pretty-formatted.yaml', 'multi-doc-not-pretty-formatted.yaml'])
def test_pretty_format_yaml_autofix(tmpdir, no_pretty_file_name):
   ...
```

Reply:
Good suggestion.  I'm happy to update.  I have very little experience with pytest, so it was mostly ignorance on my part.

Reply:
Don't be harsh on yourself ... every day is a good day to learn new things